includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -206,29 +207,29 @@ public function __construct() {
 	}
 
 	/**
-	 * Include Multibanco in the list of payment methods.
+	 * Include Multibanco in the list of payment methods when available.
 	 * As we are not registering Multibanco to show in the payment settings page and Multibanco is not a UPE payment method,
 	 * we need to include it here separately so that it's available in the checkout, pay for order, add payment method etc. pages.
 	 *
 	 * @param WC_Payment_Gateway[] $gateways A list of all available gateways on the payments settings page.
 	 * @return WC_Payment_Gateway[]          The same list if Multibanco is disabled or a list including the Multibanco method.
 	 */
 	public function get_available_payment_gateways( $gateways ) {
```

Comment:
Maybe this isn't needed with Split PE?

- We're including Multibanco in the gateways at https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2934/files#diff-e1e5e9b442c2b5d4105edbdc3787651333c1923dd623677025f72d25796900d6R417, 
- And ordering the gateways isn't supported in UPE

Reply:
Updated in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2934/commits/ed18ce5b897f0f8580d773aa04f7a31ac3fa26c6, but glad to iterate on it

Reply:
I agree that we don't need to include Multibanco in this place as we are now registering it in the other place. 

Just noticed one minor UI issue. This function was adding Multibanco to the checkout page right after the Stripe UPE (so that they are together and no other methods come in between). With split UPE all the methods are now separate. I see my checkout page like this when other methods are also enabled.

<img src="https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/1e9514c3-afe0-448e-af15-1b1548246109" width="50%" height="50%" />

Reply:
> Just noticed one minor UI issue. This function was adding Multibanco to the checkout page right after the Stripe UPE (so that they are together and no other methods come in between). With split UPE all the methods are now separate. I see my checkout page like this when other methods are also enabled.

Yup, I noticed something similar on my end, but the order is different:
<img width="300" src="https://github.com/woocommerce/woocommerce-gateway-stripe/assets/41606954/2467b961-600a-4b5f-a155-4cb85892ed97">

I'm not sure where the order is defined, but instead of addressing this, I'd propose to implement changing the display order for UPE. This functionality is in the original designs, but we phased it out for UPE to implement Split PE first. So, we could tackle both together for 8.1.0. What do you think? I'll create an issue for this if you agree.

Reply:
Sounds good @a-danae :+1:

Reply:
Hey hey! Here's the related GH issue to implement this 

- https://github.com/woocommerce/woocommerce-gateway-stripe/issues/2994